### PR TITLE
[INLONG-750] Improve master.ini and broker.ini

### PR DESCRIPTION
**Pull request by LeBW**
1. change default port to 8715 in master.ini
2. add configuration instruction for defEthName in broker.ini

### CODE REVIEW

File: inlong-tubemq/conf/master.ini [hunk 1]
```diff
@@ -19,7 +19,7 @@
 ; host address of master, required; must be configured at network card, enabled
 hostName=127.0.0.1
 ; port that master listens to, optional; default is 8715
-port=8000
+port=8715
```
Comment (gosonzhang) [inline inlong-tubemq/conf/master.ini:22]: Does the modification to 8715 have any special meaning?

Comment (LeBW) [reply inlong-tubemq/conf/master.ini:22]: As shown in the comment in Line 21, the default port is 8715. Besides, the default port in `MasterConfig` is 8715, too.
So maybe it's better to use 8715 here, so it won't confuse users.

Comment (gosonzhang) [reply inlong-tubemq/conf/master.ini:22]: Ok

**Review by gosonzhang — APPROVED** [review]
+1

**Pull request merged**